server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -1635,6 +1683,26 @@ public Builder customs(Map<String, Custom> customs) {
             return this;
         }
 
+        /**
+         * Adds a map of namespace to {@link OperatorMetadata} into the metadata builder
+         * @param operatorMetadata a map of namespace to {@link OperatorMetadata}
+         * @return {@link Builder}
+         */
+        public Builder put(Map<String, OperatorMetadata> operatorMetadata) {
```

Comment:
Is this used anywhere?

Reply:
Ah yeah, I used to call this method putOperatorState, but it was too long and it didn't match the naming of other methods we have in Metadata, we simply call `.put` with different types. Here's the usage here:

https://github.com/elastic/elasticsearch/pull/86224/files#diff-8ca6818d706ec1720e045ea8a1f4f8e4b5dda47c5610aed2bf8e7d55121d8391R129